__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -169,3 +170,140 @@ export const Light: StoryComponentType = {
+        [],
+    );
+    const [multiSelectError, setMultiSelectError] = React.useState("");
+    return (
+        <div style={{width: "400px"}}>
+            <div>
+                <form
+                    style={{display: "flex", flexDirection: "column", gap: 8}}
+                >
+                    <LabeledField
+                        label="Email"
+                        field={
+                            <TextField
+                                value={name}
+                                onChange={setName}
+                                validate={(value: string) => {
+                                    const emailRegex =
+                                        /^[^@\s]+@[^@\s.]+\.[^@.\s]+$/;
+                                    if (!emailRegex.test(value)) {
+                                        return "Please enter a valid email";
+                                    }
+                                }}
+                                onValidate={setNameError}
+                            />
+                        }
+                        error={nameError}
+                    />
+                    <LabeledField
+                        label="Description"
+                        field={
+                            <TextArea
+                                value={description}
+                                onChange={setDescription}
+                                validate={(value: string) => {
+                                    if (value.length < 5) {
+                                        return "Please enter at least 5 characters";
+                                    }
+                                }}
+                                onValidate={setDescriptionError}
+                            />
+                        }
+                        error={descriptionError}
+                    />
+                    <LabeledField
+                        {...args}
+                        label="Fruit"
+                        error={singleSelectError}
+                        field={
+                            <SingleSelect
+                                placeholder="Choose a fruit"
+                                selectedValue={singleSelectValue}
+                                onChange={(value) => {
+                                    setSingleSelectValue(value);
+                                    if (value === "strawberry") {
+                                        setSingleSelectError(
+                                            "Don't select strawberry.",
+                                        );
+                                    } else {
+                                        setSingleSelectError("");
+                                    }
+                                }}
+                                error={!!singleSelectError} // should not need to do this! LabeledField should set this
```

Comment:
LabeledField can inject the `error` prop based on if `LabeledField` receives an error message! However, this will only work if the field supports an `error` prop (so it wouldn't currently work with the TextArea/TextField components that only support the `validate`/`onValidate` props)